components/bl-radio-buttons-list-component/src/index.js

```diff
@@ -0,0 +1,58 @@
+import { useState, useEffect } from 'react';
+
+import { RadioButton } from './components';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function RadioButtonsListComponent({ component, eventHandlers }) {
+  const { classList, style, display, disabled, options, value } = component;
+  const { onChange } = eventHandlers;
+
+  const [optionsList, setOptionsList] = useState([]);
+  const [selectedValue, setSelectedValue] = useState(null);
+
+  component.getValue = () => selectedValue;
+  component.setValue = value => { setSelectedValue(value) };
+  component.getOptions = () => optionsList;
+  component.setOptions = options => { setOptionsList(options) };
+
+  useEffect(() => {
+    setOptionsList(validate(options));
+    setSelectedValue(value);
+  }, [options, value]);
+
+  const handleChange = value => {
+    setSelectedValue(value);
+
+    if (onChange) {
+      onChange({ value });
+    }
+  };
```

Comment:
```suggestion
  const handleChange = value => {
    setSelectedValue(value);
    onChange({ value });
  };
```

No need `if`. That should works without condition